common/src/main/resources/parserFieldValidValues.txt

```diff
@@ -1,3 +1,5 @@
+LEI|String with no space
```

Comment:
Since we're checking for more than just a space let's make this a bit more general. How about "String with only letters and numbers". Something like that?

Reply:
addressed